roles/upgrade/tasks/redhat-upgrade-tool-upgrade.yml

```diff
@@ -94,6 +94,8 @@
 - name: redhat-upgrade-tool-upgrade | Reboot to continue OS upgrade
   ansible.builtin.reboot:
     msg: Host is starting OS upgrade now!
+    reboot_timeout: "{{ upgrade_timeout }}"
     post_reboot_delay: "{{ post_reboot_delay }}"
-    reboot_timeout: 43200
+  timeout: "{{ upgrade_timeout }}"
```

Comment:
task keyword 'timeout' may not be present in Ansible 2.9, so this change takes infra.leapp to ansible-core 2.10+

evidence:  
see timeout task keyword in [stable-2.10](https://github.com/ansible/ansible/blob/05b90ab69a3b023aa44b812c636bb2c48e30108e/docs/docsite/keyword_desc.yml#L73) but not [stable-2.9](https://github.com/ansible/ansible/blob/5cfdbd83151edd000e79edf550f0c5bc54aebd19/docs/docsite/keyword_desc.yml#L70-L71)

Reply:
We do not support 2.9

Reply:
We should update https://github.com/redhat-cop/infra.leapp/blob/main/meta/runtime.yml.

Reply:
Do we still need this?  https://github.com/jeffmcutter/ansible-leapp/blob/main/.github/workflows/ansible-test.yml#L78